sdk/schemaregistry/schema-registry-avro/test/public/utils/dummies.ts

```diff
@@ -30,3 +31,17 @@ export const testSchemaIds = [
 export const testSchema = JSON.stringify(testSchemaObject);
 export const testValue = { name: "Nick", favoriteNumber: 42 };
 export const testAvroType = avro.Type.forSchema(testSchemaObject, { omitRecordMethods: true });
+export const testDateSchemaObject: avro.schema.RecordType = {
+  type: "record",
+  name: "AvroUser",
+  namespace: "com.azure.schemaregistry.samples",
+  fields: [
+    { name: "amount", type: "int" },
+    { name: "time", type: { type: "long", logicalType: "timestamp-millis" } },
+  ],
+};
+export const testDateSchema = JSON.stringify(testDateSchemaObject);
```

Comment:
Is it necessary to have this defined here? Seems like it is fine to inline it in its test.

Reply:
Could we also inline the transaction value?